src/Network/Network/AzureFirewallPolicy/ApplicationRuleCondition/NewAzureFirewallPolicyApplicationRuleCommand.cs

```diff
@@ -57,7 +57,6 @@ public class NewAzFirewallPolicyApplicationRuleCommand : AzureFirewallPolicyBase
 
         [Parameter(
             Mandatory = true,
-            ParameterSetName = AzureFirewallPolicyApplicationRuleParameterSets.TargetFqdn,
```

Comment:
removing this line will cause "protocol" mandatory for all parameter sets which is breaking change, please add it back

Reply:
This was a bug that I noticed and hence removed it. This parameter set applies to "FqdnTag" field and not "Protocol" field. Again even if this is a breaking change, since this is a preview product, we are fine with it.

Reply:
sorry I don't understand how is this applied to "FqdnTag", this change means protocol is now mandatory not only for "AzureFirewallPolicyApplicationRuleParameterSets.TargetFqdn" but all parameter sets.

Reply:
May be I do not understand the use of ParameterSetName field. Without my change, what does having the TargetFqdn in the ParameterSetName for "Protocol" field mean? Does it mean, if user specifies, TargetFqdn, then they have to specific Protocol but if they specific FqdnTag, then they should not specific Protocol?

Reply:
Got clarification from Anton and hence reverted his change.